version/version.go

```diff
@@ -44,7 +44,11 @@ func getSDKVersion() string {
 	var sdkVersion string
 	for _, dep := range deps.Deps {
 		if dep.Path == "github.com/cosmos/cosmos-sdk" {
-			sdkVersion = dep.Version
+			if dep.Replace != nil {
```

Comment:
```suggestion
			if dep.Replace != nil && dep.Replace.Version != "(devel)" {
```

I do not think showing "(devel)" is useful, this way when using a local replace you can still set an arbitrary version in the go.mod to display a proper version. We use this trick in simapp:  https://github.com/cosmos/cosmos-sdk/blob/506e87a/simapp/go.mod#L23-L24